app/js/controllers/rules_controller.js

```diff
@@ -6,12 +6,16 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
 
   $scope.rule_id = parseInt($routeParams.id, 10);
   $scope.pr_ch_options = ["All rules"];
-  if ($scope.rule_id) {
-    // history of a specific rule
-    Rules.getHistory($scope.rule_id)
+
+  $scope.currentPage = 1;
+  $scope.pageSize = 10;
+  $scope.maxSize = 5;
```

Comment:
We've generally got lots of width around the pagination controls - can we raise this to 10 to make it a bit easier to browse around?